html.md

```diff
@@ -1,68 +1,83 @@
 # HTML
 
- 1. **[Attribute Ordering](#attribute-ordering-within-tags)**
+## Table of Contents
+ 1. **[Attribute Ordering](#attribute-ordering-within-tags)** [ [Attribute Order](#11-attribute-order) ]
  1. **[Quotes](#quotes)**
- 1. **[Buttons](#buttons)**
+ 1. **[Inputs](#inputs)**
+ 1. **[Buttons](#buttons)** [ [Submit Buttons](#31-submit-buttons) ] [ [Links](#32-links) ] [ [Buttons](#33-buttons) ]
  1. **[Forms](#forms)**
- 1. **[Comments](#comments)**
- 1. **[Semantics](#semantics)**
+ 1. **[Comments](#comments)** [ [When Things Get Nesty](#51-when-things-get-nesty) ]
+ 1. **[Semantics](#semantics)** [ [`<strong>` vs `<b>`](#61-strong-vs-b) ] [ [`<em>` vs `<i>`](#62-em-vs-i) ]
  1. **[Do NOT Use](#do-not-use)**
 
-<a name="attribute-ordering-within-tags"></a>
-## 1. Attribute Ordering within Tags
+
```

Comment:
why did you remove the anchor link?

Reply:
Line 11 was gone in the CR

Reply:
Correct. @ASH-Bryan tried to point this out in a recent code review. The anchor links are not needed and add clutter. You can target the headings themselves.